File: python/translator.py
import requests

SERVER = "http://localhost:8888"

def translate():
    text = input("Enter error message: ")
    try:
        r = requests.get(f"{SERVER}/translate", params={"q": text})
        print(r.json())
    except:
        print("[ERROR] Server not running qwq")

if __name__ == "__main__":
    translate()

File: js/translator.js
const http = require('http');
const readline = require('readline');

const rl = readline.createInterface({ input: process.stdin, output: process.stdout });

rl.question('Enter error message: ', (input) => {
    http.get(`http://localhost:8888/translate?q=${encodeURIComponent(input)}`, (res) => {
        let data = '';
        res.on('data', chunk => data += chunk);
        res.on('end', () => { console.log(data); rl.close(); });
    }).on('error', () => {
        console.log('[ERROR] Server not running qwq');
        rl.close();
    });
});